src/py/flwr/common/constant.py

```diff
@@ -87,6 +87,9 @@
 ISOLATION_MODE_SUBPROCESS = "subprocess"
 ISOLATION_MODE_PROCESS = "process"
 
+# Log streaming
+LOG_UPLOAD_INTERVAL = 0.2
```

Comment:
Do we want to make 5 requests per second? Every second might be sufficient, maybe even every other second.

Reply:
Hmmm, we won't really make 5 requests per second. If no new message, it won't make a request. It's actually the minimum interval for log uploading.